dataeng/jobs/analytics/PrefectFlowsDeployment.groovy

```diff
@@ -0,0 +1,162 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_authorization
+import static org.edx.jenkins.dsl.AnalyticsConstants.slack_publisher
+
+
+class PrefectFlowsDeployment{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("prefect-flows-deployment-poll"){
+            authorization common_authorization(allVars)
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('PREFECT_FLOWS_URL', allVars.get('PREFECT_FLOWS_URL'), 'URL for the prefect-flows repository.')
+                stringParam('PREFECT_FLOWS_BRANCH', allVars.get('PREFECT_FLOWS_BRANCH'), 'Branch of prefect-flows repository to use.')
+                stringParam('JENKINS_JOB_DSL_URL', allVars.get('JENKINS_JOB_DSL_URL'), 'URL for the jenkins-job-dsl repo.')
+                stringParam('JENKINS_JOB_DSL_BRANCH', allVars.get('JENKINS_JOB_DSL_BRANCH'), 'Branch of jenkins-job-dsl repo to use.')
+            }
+            scm {
+                git {
+                    remote {
+                        url('$PREFECT_FLOWS_URL')
+                        branch('$PREFECT_FLOWS_BRANCH')
+                        credentials('1')
+                    }
+                    extensions {
+                        relativeTargetDirectory('prefect-flows')
+                        pruneBranches()
+                        cleanAfterCheckout()
+                    }
+                }
+
+            }
+            triggers {
+                scm('H/2 * * * *') // change it to 10 minutes after testing
+            }
```

Comment:
For few days of launch let it poll 2 minutes to help in testing quickly then change it to 10 minutes or 15 minutes

Reply:
Makes sense. In general, we put cron statements in to the JOB_CONFIGS, so maybe we should do the same here.